bin/ansible2gdeploy.py

```diff
@@ -24,36 +25,43 @@
 
 def main():
     ap = argparse.ArgumentParser(
-        description="in place update of hosts in gdeploy conf file")
+        description="update of hosts and devices in gdeploy conf file")
     ap.add_argument(
         "-d",
         "--dry-run",
         action="store_true",
-        help="print to stdout instead of in place edit")
-    ap.add_argument("gdeployconf", help="gdeploy config file to edit")
+        help="print to stdout instead of save to file")
```

Comment:
We no longer do in place edit?

Reply:
You can use *in place* edit - by omitting `-p` (output file prefix) argument.